### DISCUSSION TITLE
Is it possible to shorten the import of station prices via edmc_batch?

### DISCUSSION THREAD

**Discussion by nepomuk16321** [eyeonus/Trade-Dangerous#173] [Q&A] [answered]
OK, next question/statement. When I perform a station import via `edmc_batch`, I get a warning.
And the end of the command takes quite a long time (cursor flashes again), although the message "Import complete"
has been displayed for several seconds.
![image](https://github.com/eyeonus/Trade-Dangerous/assets/20616775/500773ba-b166-4a04-9b1f-0a9b0fbbd619)

**Comment by eyeonus** [comment, marked as answer]
It's regenerating the prices file `TradeDangerous.prices` to include the new data just imported, and that step doesn't output anything unless `-w` is used, which is why it seems to not be doing anything.

**Reply by nepomuk16321** [reply]
Ok, I specified -w on the last import and TD showed #Regenerating .prices file as the last action.
So that's what the time is needed for. Thank you for the info, @eyeonus  !
![image](https://github.com/eyeonus/Trade-Dangerous/assets/20616775/051778ee-45fa-4b46-ac03-72791f890436)
(sorry for the background color)
